tests/test_errorcallback.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the error callback"""
+
+import unittest
+import unittest.mock
```

Comment:
NB: if you don't have Python 3.3, 3.4 or 3.5, you can call tox with a single test environment, e.g. `tox -e py34`.